### PR TITLE
Cronjob to delete old metric entries

### LINKED ISSUE #57: Cronjob to delete old metric entries
Description:
- Delete entries before 3 months in a Lambda
- Mimic the snapshot lambda (see Terraform and snapshot lambda in the `/cmd` folder)

Estimation: 3

### PR DESCRIPTION
**Pull request by frankhillard**

### CODE REVIEW

**Comment by aboulay** [comment]
[WFD] Wait for dependancies on infrastructure : https://github.com/octo-technology/tezos-link/pull/51

File: cmd/metrics/main.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/aws/aws-lambda-go/lambda"
+	pkgdatabase "github.com/octo-technology/tezos-link/backend/pkg/infrastructure/database"
+	"log"
+	"os"
+)
+
+//var configPath = flag.String("conf", "", "Path to TOML config")
```
Comment (aboulay) [inline cmd/metrics/main.go]:
```suggestion

```

File: cmd/metrics/main.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+	err = con.Ping()
+	if err != nil {
+		log.Fatal("Could not open DB: ", err)
+	}
+
+	log.Println("DB initialized")
+	Connection = con
+
+/*	log.Print("DB URL", config.ProxyConfig.Database.Url)
```
Comment (aboulay) [inline cmd/metrics/main.go]: Deadcode here ?

**Review by aboulay — CHANGES_REQUESTED** [review]
Just update those two short comment and we are fine ! :D Well done !

File: cmd/metrics/main.go (hunk 1 above)
Comment (frankhillard) [reply cmd/metrics/main.go]: comment deleted

File: cmd/metrics/main.go (hunk 2 above)
Comment (frankhillard) [reply cmd/metrics/main.go]: comment deleted

**Review by aboulay — APPROVED** [review]
Well done !

**Pull request merged**